config/crd/bases/policies.kubewarden.io_admissionpolicies.yaml

```diff
@@ -163,11 +163,13 @@ spec:
                           items:
                             type: string
                           type: array
+                          x-kubernetes-list-type: atomic
```

Comment:
Where is this coming from?

Reply:
This came from a recent [update](https://github.com/kubernetes/apimachinery/commit/bfbbdecc1d6d161ce84da90c4a58e21c6eb8261f) in the apimachinery package. As we use label selector objects in the policy spec, the CRDs will be updated with the latest changes there as well. I've added a commit here to fix that in our main branch and avoid generating noise every time that we run the controller locally.  

If you think is better, I can open another PR just for that update in the CRDs.